projects/redis/redis/docker/linux/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE
+ARG BUILDER_IMAGE
+FROM $BUILDER_IMAGE as redis-builder
+
+COPY redis /redis
+RUN set -x && yum install -y gcc make
+WORKDIR /redis
+RUN case "$(uname -p)" in \
```

Comment:
Another option to this would be just to use the redis provided by al2. I suspect its really out of date so may not be the best option, but maybe when al2022 is out we could rely on the redis provided there and just stay aligned with that?  That would sorta push off some of the maintenance from us since we could rely on them to keep it up to date and backport cves and what not. Just a thought.

Reply:
Darn, I didn't know they had it and v6.2.6 which is the latest. I'll switch to that.

Reply:
Switched to AL2 redis